SpatialGDK/Source/SpatialGDKFunctionalTests/Public/TestWorkerSettings.h

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "LoadBalancing/GridBasedLBStrategy.h"
+#include "LoadBalancing/SpatialMultiWorkerSettings.h"
+
+#include "TestWorkerSettings.generated.h"
+
+/**
+ * A selection of load balancing strategies for testing zoning features.
+ */
+
+/**
+ * Full interest grid strategies
+ * The strategies in this group have a world-wide interest border, so everything should be in view.
+ * UTest1x2FullInterestGridStrategy
+ * UTest2x1FullInterestGridStrategy
```

Comment:
This would be for the later ticket (reduce number of test maps by merging them), but 1x2 and 2x1 don't really have any functional difference, and tests written for one could be converted to use the other.

Reply:
True, though I somewhat prefer actually having both available, though it would've been nicer if we could actually test a variety of suitable LB configs per test (during a nightly for example).